frontend/src/vault/deposit/DepositVerify/utils.ts

```
@@ -12,6 +14,12 @@ export const getFormattedFormData = (
 
     if (!value) return;
 
+    // For THORChain leave we need to hardcode the amount to 0.00000001 RUNE for the transaction
+    if (chainAction === 'leave') {
+      const runeDecimals = getChainFeeCoin(Chain.THORChain)?.decimals || 8;
```

No need for  || 8, it should throw an error if nothing is found